pvlib/scaling.py

```diff
@@ -0,0 +1,320 @@
+"""
+The ``scaling`` module contains functions for manipulating irradiance
+or other variables to account for temporal or spatial characteristics.
+"""
+
+import numpy as np
+import pandas as pd
+
+
+def wvm(cs_series, latitude, longitude, cloud_speed,
+        method, capacity=None, density=41, dt=None):
```

Comment:
I looked into the `method` meaning. Basically, it's a convenience to communicate a polygon (or simple square) in lieu of a list of points. In MATLAB, a polygon or square is gridded at some resolution to create a list of points that cover the area, after which the algorithm is the same as if `discrete` is provided.

My vote here is to remove `method` and the kwargs supporting the 'square' option (`capacity` and `density`). In a future PR we can implement a public function `prepare_wvm_inputs` that accepts the simpler inputs (e.g. a polygon) and returns the list of grid points. That keeps the `wvm` function with a single purpose: compute the smoothed clearsky index using the wavelet transform over a grid of points.

Would appreciate other's views here @wholmgren @mikofski

Reply:
From my end, I like this suggestion. Wrapping these together was a holdover from Matlab, but given that it's being done in a more compartmented way here, I think keeping this function simple is a better choice.

Reply:
I agree.